docs/.gitignore

```diff
@@ -0,0 +1 @@
+_build/*
```

Comment:
by the way: why are the local builds of the doc not ignored?

if we want to have a "local" version, a PDF may be better, no?

Reply:
I am adding this gitignore.

But I am asking permission to nuke the _build from the repo.

Reply:
Probably just me not being attentive... I agree that we can kick it out.

Reply:
done